examples/platform/silabs/uart.cpp

```diff
@@ -411,7 +411,7 @@ static void UART_rx_callback(UARTDRV_Handle_t handle, Ecode_t transferStatus, ui
 
 #ifdef ENABLE_CHIP_SHELL
     chip::NotifyShellProcess();
-#elif !defined(PW_RPC_ENABLED)
+#elif !defined(PW_RPC_ENABLED) && CHIP_DEVICE_CONFIG_THREAD_ENABLE_CLI == 1
```

Comment:
can we use just `CHIP_DEVICE_CONFIG_THREAD_ENABLE_CLI` ? since it seems to be a 0/1 flag

Reply:
I can. I felt == X protects againts undefined values in case this doesn't get defined. This one is always define by `CHIPConfig.h`. I'll change it